ReservacionDeVuelos/src/screens/LogInScreen.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import {View, Text, StyleSheet, TouchableOpacity} from 'react-native';
+
+const LogInScreen = ({navigation}) => {
+  return (
+    <View style={styles.container}>
+      <Text style={styles.text}>LOG IN SCREEN</Text>
+      <TouchableOpacity onPress={() => navigation.navigate('SignIn')}>
+        <Text style={styles.goToSignIn}>
+          Don't have an account? Create here!
```

Comment:
could you create a constants.js file, that is to add the text LOG IN SCREEN and Don't have an account? Create here!, so you can import it wherever you need it. 

example constants.js
const LOGIN_SCREEN = 'LOG IN SCREEN'; 
CONST NO_ACCOUNT = 'Don't have an account? Create here!'